DataFormats/GsfTrackReco/src/classes_def.xml

```diff
@@ -25,7 +25,7 @@
   <class name="edm::RefVector<std::vector<reco::GsfTrackExtra>,reco::GsfTrackExtra,edm::refhelper::FindUsingAdvance<std::vector<reco::GsfTrackExtra>,reco::GsfTrackExtra> >"/>
 
   <class name="reco::GsfTrack" ClassVersion="20">
-   <version ClassVersion="20" checksum="3090385640"/>
+   <version ClassVersion="20" checksum="866972163"/>
```

Comment:
the errors are from here; a new version is needed, updates of checksums to existing production versions are verboten